src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -750,6 +758,14 @@ function build_eal_list() {
 	    'to allow connections from any IPv6 address.' . '<br/>' . 'Responder Only must be set and ' . 
 	    'Peer IP Address cannot be used for Remote Identifier.'), 'info', false),
 	    '</div>');
+	$group->add(new Form_Input(
+    'ikeport',
+		'Remote IKE Port',
+		'number',
+		$pconfig['ikeport'],
+		['min' => 1, 'max' => 65535]
+	))->setHelp('Enter a custom IKE port number for IPsec (leave empty for 500/4500 default scheme)');
```

Comment:
This could be: `UDP port for IKE on the remote gateway. Leave empty for default automatic behavior (500/4500)`